dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/datafile/page/ReorderDataFilesPage.java

```diff
@@ -32,6 +34,8 @@
     private DatasetVersion datasetVersion = new DatasetVersion();
     private List<FileMetadata> fileMetadatas;
     private List<FileMetadata> fileMetadatasCopy;
```

Comment:
Suggestion: I think that this variable could be renamed, as it is not, in fact, a copy, but another reference to an object.

Reply:
How about:
`fileMetadatas` -> `fileMetadatasOriginal`
`fileMetadatasCopy` -> `fileMetadatasReordered`?

Reply:
It would certainly be better, however I was thinking about some more "technical" name, like ```fileMetadatasRef``` – because, before I spotted assignment in the ```init``` method, I was puzzled why in one method ```filesMetadataCopy``` is updated, and in the other not, so in this case including info, that the variable is referring some existing object, in its name would be IMHO helpful.

Reply:
TBH I can't reproduce any null's that supposedly happens when using `filesMetadataCopy` instead of `filesMetadata`. Maybe it doesn't happen now, and we could get rid of `filesMetadataCopy` altogether

Reply:
If so, then getting rid of this variable would be the best idea.